### ISSUE TITLE
vm creation - rename action

### ISSUE THREAD

**Issue by ronenpi18** [duboc/vra-repo#1]
hey, 
I know it is not exactly an issue about your repo, but I'm working on a project (VMware partner) that uses some of your actions and I have a few questions... 
I have looked all over the internet for the past 4 weeks and can't find an answer. 

I have a simple multiple VMs creation at my vRA 8.1, somehow it overrides the names I chose for my VMs, and puts some random stuff instead.

can we contact on hangouts or zoom or any other telecommunications method?
please! 
I have to get it right... it's driving me crazy...

I beg for help, from expert like you.

Ronen.

**Comment by duboc** [comment]
of course. ping me on telegram or twitter @duboc and I can get this right with you.
